src/RequestHandler/inprocess/inprocessapplication.cpp

```diff
@@ -862,14 +627,15 @@ IN_PROCESS_APPLICATION::ExecuteApplication(
 // Calls hostfxr_main with the hostfxr and application as arguments.
 // Method should be called with only 
 // Need to have __try / __except in methods that require unwinding.
+// Note, this will not 
 // 
 HRESULT
-IN_PROCESS_APPLICATION::RunDotnetApplication(PCWSTR* argv, hostfxr_main_fn pProc)
+IN_PROCESS_APPLICATION::RunDotnetApplication(CONST DWORD argc, PCWSTR* argv, hostfxr_main_fn pProc)
```

Comment:
do you want to trust the passed in argc matching with the counter of argv

Reply:
I think it's a valid assumption.